### PR TITLE
[WIP] ath79: add alternative device names

### PR DESCRIPTION
**Pull request by mwarning**
Add more DEVICE_ALT0_* fields to make it easier
to find the models using the firmware selector.

Signed-off-by: Moritz Warning <moritzwarning@web.de>

### CODE REVIEW

**Comment by mwarning** [comment]
@adschm is this a good idea? I will add more data as many TP-Link models have alternative names.
The inspiration comes from here: https://forum.openwrt.org/t/the-openwrt-firmware-selector/81721/10

File: target/linux/ath79/image/generic-tp-link.mk [hunk 1]
```diff
@@ -569,6 +575,9 @@ define Device/tplink_tl-wdr7500-v3
   SOC := qca9558
   DEVICE_MODEL := TL-WDR7500
   DEVICE_VARIANT := v3
+  DEVICE_ALT0_VENDOR := TP-Link
+  DEVICE_ALT0_MODEL := Archer C7R
```
Comment (mwarning) [inline target/linux/ath79/image/generic-tp-link.mk:579]: Maybe this should be swapped with DEVICE_MODEL.

**Comment by adschm** [comment]
I never liked ALTx too much because it's limited to `make menuconfig`. Apart from that, I don't see any value in adding the TP-Link ACxxxx names, as they are quite unspecific.

For the C7R it appeared to me that the situation is complicated, I'm not sure whether this is really exactly the same as TL-WDR7500 v3. And even if it was, I'd probably prefer full separate images as the names aren't even roughly similar.

So, sorry, but I probably would rather not like this merged.

**Comment by mwarning** [comment]
Ok.

**Pull request closed without merging**